pal/system-info/minigame/system-info.ts

```diff
@@ -97,12 +97,17 @@ class SystemInfo extends EventTarget {
         this.isXR = false;
 
         // init capability
+        const isIOSWechat = WECHAT && this.os === OS.IOS && !minigame.isDevTool;
```

Comment:
```suggestion
        // NOTE: canvas.toDataURL() is not supported on WeChat iOS end
        const isIOSWechat = WECHAT && this.os === OS.IOS && !minigame.isDevTool;
```

It's better to add some comment why we do this bug fix